src/sensors/CyclePowerData.cpp

```diff
@@ -10,20 +10,15 @@
 
 bool CyclePowerData::hasHeartRate() { return false; }
 
-bool CyclePowerData::hasCadence() { return this->cadence > 0 || this->missedReadingCount < 3; }
+bool CyclePowerData::hasCadence() { return !isnan(this->cadence); }
 
-bool CyclePowerData::hasPower() { return this->power != INT_MIN; }
+bool CyclePowerData::hasPower() { return true; }
```

Comment:
Why change this to indicate that there is power before a `decode` pass?

Reply:
To be honest, I was just matching `EchelonData.cpp`. Should `EchelonData.cpp` do the same, or is there a different situation for that one?